gaphor/storage/storage.py

```diff
@@ -67,8 +67,7 @@ def save_element(name, value, writer):
 
     A value may be a primitive (string, int), a
     gaphor.core.modeling.collection (which contains a list of references
-    to other UML elements) or a gaphas.Canvas (which contains canvas
```

Comment:
With this change does the concept of a `canvas item` still exist in Gaphor?

Reply:
Should be updated